### DISCUSSION TITLE
Hello there is a way to deactivate some shorcuts?

### DISCUSSION THREAD

**Discussion by GabMesa** [Happy-Coding-Clans/vue-easytable#586] [Q&A] [answered]
I'm working on inputs inside the table and the backspace do not work properly on the inputs because the event of the cell collides with the input inside. I want to deactivate the shortcuts of the cell, and be able to select cells. It is there a way to do that?

**Comment by huangshuwei** [comment, marked as answer]
You can disable the cell selection function, so all shortcut keys will be disabled.

```
cellSelectionOption: {
       // default true
       enable: false,
},
```

Here is the example:https://happy-coding-clans.github.io/vue-easytable/#/en/doc/table/cell-edit?anchor=combine-element-ui

**Comment by lzj130** [comment]
Can disabling a certain cell be achieved?
